src/components/cart/CartToggle/cart-toggle.scss

```diff
@@ -0,0 +1,39 @@
+[data-store-button].cart-toggle {
+    cursor: pointer;
+
+    background-color: #FFF0;
+    border-width: 0;
+    position: relative;
+
+    svg {
+      width: var(--space-5);
+      height: var(--space-5);
+    }
+
+    &::after {
+      content: attr(data-items);
+
+      position: absolute;
+      top: var(--space-0);
+      left: var(--space-4);
+
+      font-weight: var(--text-weight-bold);
+      font-size: var(--text-size-0);
+      color: var(--color-text-negative);
+
+      background: var(--bg-secondary-default);
+      border-radius: var(--border-radius-pill);
+      padding: calc(var(--space-0)/4) var(--space-0);
```

Comment:
I think we can create a new variable for `1px` or just use `1px` here.

Reply:
Just use 1px here! 

How about we follow this approach?
If it is less than 4px, just use it.